src/main/java/com/kodilla/ecommercee/domain/user/UserEntity.java

```diff
@@ -37,6 +38,7 @@ public UserEntity(String name, String surname, String login, String mail, String
         this.login = login;
         this.mail = mail;
         this.password = password;
+        this.loggedIn = false;
```

Comment:
dodałeś to z powodu funkcjonalności - blokada użytkownika?

Reply:
Nie z powodu konieczności wygenerowania klucz ważnego godzinę w jenum z endpointór najlepiej według speca w pierwszym module kodilli.